rust/src/lib/iface.rs

```diff
@@ -315,6 +315,9 @@ impl<'de> Deserialize<'de> for Interface {
             if let Some(s) = v.get("mac-address") {
                 new_value.insert("mac-address".to_string(), s.clone());
             }
+            if let Some(s) = v.get("pci-address") {
+                new_value.insert("pci-address".to_string(), s.clone());
+            }
```

Comment:
Some lines above these ones there is a comment `// Ignore all properties except type if state: absent`. It's clearly outdated, please change it.
Apart from this, LGTM.